### PR TITLE
[#3179]fix Notify's typos in common module and remove NotifyCenter's unit test in test module

### PR DESCRIPTION
**Pull request by zongtanghu**
Please do not create a Pull Request without creating an issue first.

## What is the purpose of the change

fix Notify's typos in common module and remove NotifyCenter's unit test in test module

## Brief changelog

XX

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

* [x] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
* [x] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
* [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
* [x] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/alibaba/nacos/tree/master/test).
* [x] Run `mvn -B clean package apache-rat:check findbugs:findbugs -Dmaven.test.skip=true` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.

### CODE REVIEW

**Comment by zongtanghu** [comment]
In this pr, I have done two points, firstly:

(1). Fix some typo which @chuntaojun give some suggestion in previous closed pr.(The pr link is : https://github.com/alibaba/nacos/pull/3180)
(2). Remove Original Notify unit test case codes which have already been moved to nacos's common module.

**Comment by zongtanghu** [comment]
Please help to view this pr, thanks. @chuntaojun @KomachiSion

File: common/src/main/java/com/alibaba/nacos/common/notify/NotifyCenter.java [hunk 1]
```diff
@@ -46,9 +47,9 @@
     
     private static final Logger LOGGER = LoggerFactory.getLogger(NotifyCenter.class);
     
-    public static int ringBufferSize = 16384;
+    public static int RING_BUFFER_SIZE = 16384;
```
Comment (KomachiSion) [inline common/src/main/java/com/alibaba/nacos/common/notify/NotifyCenter.java:50]: No `static final` use camel naming. Your changes can't pass checkstyle

Comment (zongtanghu) [reply common/src/main/java/com/alibaba/nacos/common/notify/NotifyCenter.java]: I hava alreayd fix this issue in next commit.

**Pull request merged**